openstack/networking/v2/subnets/requests.go

```diff
@@ -199,8 +199,7 @@ type UpdateOpts struct {
 	// AllocationPools are IP Address pools that will be available for DHCP.
 	AllocationPools []AllocationPool `json:"allocation_pools,omitempty"`
 
-	// GatewayIP sets gateway information for the subnet. Setting to nil will
-	// cause a default gateway to automatically be created. Setting to an empty
+	// GatewayIP sets gateway information for the subnet. Setting to an empty
```

Comment:
>Setting to an empty will cause the subnet to be created with no gateway

Maybe since we're here we can rephrase: this is `UpdateOpts` and we are not creating